proposals/nnnn-async-await.md

```diff
@@ -17,7 +17,7 @@ This proposal defines the semantics of asynchronous functions. However, it does
 This proposal draws some inspiration (and most of the Motivation section) from an earlier proposal written by 
 [Chris Lattner](https://github.com/lattner) and [Joe Groff](https://github.com/jckarter), available [here](https://gist.github.com/lattner/429b9070918248274f25b714dcfc7619). That proposal itself is derived from a proposal written by [Oleg Andreev](https://github.com/oleganza), available [here](https://gist.github.com/oleganza/7342ed829bddd86f740a). It has been significantly rewritten (again), and many details have changed, but the core ideas of asynchronous functions have remained the same.
 
-Swift-evolution thread: [Discussion thread topic for that proposal](https://forums.swift.org/t/concurrency-asynchronous-functions/41619)
+Swift-evolution thread: [\[Concurrency\] Asynchronous functions](https://forums.swift.org/t/concurrency-asynchronous-functions/41619)
```

Comment:
```suggestion
Swift-evolution thread: [Discussion thread topic for that proposal](https://forums.swift.org/t/concurrency-asynchronous-functions/41619)
```

to be consistent with all other proposal documents

Reply:
@ktoso "Discussion thread topic for that proposal" is from the [template](https://github.com/apple/swift-evolution/blob/main/proposal-templates/0000-swift-template.md), but most of the accepted proposals have changed it to the actual name of their forums topic.

Reply:
I see, let's fix it in all proposals then :) 

also https://github.com/DougGregor/swift-evolution/blob/structured-concurrency/proposals/nnnn-structured-concurrency.md and the actors one.

do you want to send in the PRs?

Reply:
> do you want to send in the PRs?

Yes, I'll include the changes in my other PRs.